pkg/tgzutils/tgz.go

```diff
@@ -0,0 +1,61 @@
+package tgzutils
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+)
+
+// Uncompress decompresses a .tgz file into a directory.
+func Uncompress(tgz, dst string) error {
```

Comment:
why 'uncompress' and not 'decompress'?

'untar' would work too

'uncompress' just sounds wrong to my ear

Reply:
this would be a bit simpler if it took a stream as input 😄

Reply:
For the use case this is fine. We can always break this into two functions later and make one operate with streams instead. Renamed it to `Decompress`.